index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+      integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+      crossorigin="anonymous"
+    />
+  </head>
+
+  <body>
+
+      <div class="home-page">
+        <!-- navigation bar -->
+        <header id="header">
+
+            <div class="logo">
+              <img
+                id="header-img"
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+                alt="original trombones logo"
+              />
+            </div>
+        
+            <nav id="nav-bar">
+              <ul>
+                <li><a class="nav-link" href="#features">Features</a></li>
+                <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+                <li><a class="nav-link" href="#pricing">Pricing</a></li>
+              </ul>
+            </nav>
+        </header>
+
+        <!-- Hero section -->
+        <section id="hero">
+            <h2>Handcrafted, home-made masterpieces</h2>
+            <form id="form" action="https://www.freecodecamp.com/email-submit">
+              <input
+                name="email"
+                id="email"
+                type="email"
+                placeholder="Enter your email address"
+                required
+              />
+              <input id="submit" type="submit" value="Get Started" class="btn" />
+            </form>
+          </section>
+
+          <!-- Feature section -->
+  <div class="container">
+    <!-- <ul>
+      <li>Feature</li>
+      <li>home page</li>
+    </ul> -->
```

Comment:
Do not use unused code as a comment in your codebase. Unused code as comments makes your code not clean

Reply:
- [ ] OPTIOANL: Nice inline comment. I would recommed using more polite phrases like `It would be nice if you ...` or `I think this should be ...` which makes your comments more friendly. :pray: